test/ajv-schemes.js

```diff
@@ -0,0 +1,141 @@
+module.exports = schemesV3_0_0;
```

Comment:
I would take advantage of the existing code in the bundler that loads the definitions, bindings and extensions: https://github.com/asyncapi/spec-json-schemas/blob/8ab1e6689c7cf1a3e8bf56d30dfb7f787c642a3d/tools/bundler/index.js#L107

This implementation is gonna be annoying long term 😅

Reply:
Can you show valid example?

Reply:
Moved up to`TestHelper` class

Reply:
I can optimize it after